### PR TITLE
options: set 'display' to "lastline" by default.

### PR DESCRIPTION
**Pull request by fmoralesc**
Re: https://github.com/neovim/neovim/issues/2676

### CODE REVIEW

File: runtime/doc/options.txt [hunk 1]
```diff
@@ -2253,7 +2253,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	security reasons.
 
 					*'display'* *'dy'*
-'display' 'dy'		string	(default "")
+'display' 'dy'		string	(default "lastline", Vi default: "")
```
Comment (justinmk) [inline runtime/doc/options.txt]: I am curious, why mention "Vi default:"? I think we typically remove this from the docs.

Comment (fmoralesc) [inline runtime/doc/options.txt]: Since we still have `set ...&vi`, and the values differ, I think it's better to document it.

Comment (justinmk) [inline runtime/doc/options.txt]: Oh, good point.

**Comment by justinmk** [comment]
Merged. (Travis error is a gcov error.)

**Pull request closed without merging**